src/Faker/Provider/ng_NG/Person.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+// data source
+
+namespace Faker\Provider\ng_NG;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{lastName}}'
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}}'
+    );
+
+    /**
+     * An array of typical Nigerian male firstnames.
+     * @link https://andela-celisha-wigwe.github.io/names.html
```

Comment:
There is no open-source License [on your content](https://github.com/andela-celisha-wigwe/andela-celisha-wigwe.github.io). Faker can only reuse it if the content uses MIT or a compatible license.